gssapi/sec_contexts.py

```diff
@@ -32,11 +35,13 @@ def __new__(cls, base=None, token=None,
 
         if token is not None:
             base = rsec_contexts.import_sec_context(token)
+        elif rchannel_bindings is not None and base is None:
+            base = rchannel_bindings.create_sec_context()
 
         return super(SecurityContext, cls).__new__(cls, base)
 
-    def __init__(self, base=None, token=None,
-                 name=None, creds=None, lifetime=None, flags=None,
+    def __init__(self, base=None, token=None, name=None, creds=None,
+                 lifetime=None, flags=None, ret_flags_understood=None,
```

Comment:
we *probably* should avoid changing argument order (since this isn't python 3, we can't do keyword-only args).